pkg/lib/plot.css

```diff
@@ -11,19 +11,6 @@
     color: black;
 }
 
-.flot-y-axis .flot-tick-label {
```

Comment:
I think all of plot.css can be removed, actually.  We should have done that in #14913, but totally forgot about cleaning up properly in all the excitement...

Reply:
I didn't dare to do that yet, as `git grep plot.css` shows that 5 other files import it still.

Reply:
I see, let's do that in a separate PR then.